Tests/Specs/transactions/tests/abort.json

```diff
@@ -0,0 +1,621 @@
+{
```

Comment:
we generally commit the JSON straight to master ahead of time to reduce the diff in the PR with the actual test / code changes.

Reply:
Ok, I'll go ahead and commit the JSON for the convenient api tests straight to master.